llama_index/readers/mongo.py

```diff
@@ -48,6 +48,7 @@ def load_data(
         db_name: str,
         collection_name: str,
         field_names: List[str] = ["text"],
+        field_names_metadata: List[str] = [],
```

Comment:
this should be `field_names_metadata: Optional[List[str]] = None` and then do `field_names_metadata = field_names_metadata or []` below. safer creation of empty list